indico/modules/users/forms.py

```diff
@@ -116,6 +116,10 @@ class AdminUserSettingsForm(IndicoForm):
                                          description=_('Whether users are allowed to generate personal API tokens. '
                                                        'If disabled, only admins can create them, but users will '
                                                        'still be able to regenerate the tokens assigned to them.'))
+    if config.data['LOCAL_MODERATION']:
+        mandatory_fields_account_request = IndicoSelectMultipleCheckboxField(_('Mandatory fields in account request'),
+                                                                         choices=['Affiliation','Comment'],
```

Comment:
I'm pretty sure you can pass a list of tuples here to separate value/title (value should be lowercase, title should be translatable)